src/Utility/Functions.h

```diff
@@ -17,6 +17,9 @@
 #include "Function.h"
 
 
+class TensorFunc;
```

Comment:
Not needed, this is forward-declared in Function.h already

Reply:
removed